src/amo/reducers/users.js

```diff
@@ -288,6 +370,7 @@ const reducer = (
       return {
         ...state,
         ...addUserToState({ state, user }),
+        ...setUserAsNotEditing({ state, userId: user.id }),
```

Comment:
I'm not sure this makes sense. We're assuming that the fact that we're loading a user account means that we're no longer editing. Would it be better to have a `FINISH_EDIT_USER_ACCOUNT` that is explicitly called when we know the edit is finished? That's what I did in #4723, but maybe that's not the best approach.

Oh! Is that what `CANCEL_EDIT_USER_ACCOUNT` is all about?

Reply:
We *could* have a separate action, I'm fine with that. Cancel was for errors, but I guess I could just change it to `FINISH` and use it in either place.

Reply:
Yeah, now I understand about Cancel. I called it Finish in my patch. I find the term `Cancel` confusing because it sounds like you're actually cancelling (i.e., aborting) something, when really it's just about updating the `editing` state. I would prefer that we use Begin and Finish in both patches, if that's okay with you.

Reply:
Finish is totally better, wfm 👍